src/Files.App/Controllers/SidebarPinnedController.cs

```diff
@@ -163,7 +175,8 @@ public void SaveModel()
 			}
 		}
 
-		private async Task<IEnumerable<string>> ReadV1PinnedItemsFile()
+		[Obsolete]
```

Comment:
Is this `[Obsolete]` tag suggested by your VS Code?
Not sure this is useful here?